plugins/workspace-minimap/README.md

```diff
@@ -30,9 +52,33 @@ const minimap = new Minimap(workspace);
 minimap.init();
 ```
 
+```css
+.blockly-minimap {
+  position: absolute;
+  box-shadow: none;
+  width: 200px;
+  height: 150px;
+  top: 0px;
+  left: 50vw;
+}
+```
+
+### Configuration
+The minimap takes a workspace as input and it inherits its RTL and theme properties (so that they don't need ot be configured manually).
```

Comment:
```suggestion
The minimap takes a workspace as input and it inherits its RTL and theme properties (so that they don't need to be configured manually).
```